engine/schema/src/main/resources/META-INF/db/schema-41520to41600.sql

```diff
@@ -791,3 +791,6 @@ ALTER TABLE cloud.user_vm_details MODIFY value varchar(5120) NOT NULL;
 ALTER TABLE cloud_usage.usage_network DROP PRIMARY KEY, ADD PRIMARY KEY (`account_id`,`zone_id`,`host_id`,`network_id`,`event_time_millis`);
 ALTER TABLE `cloud`.`user_statistics` DROP INDEX `account_id`, ADD UNIQUE KEY `account_id`  (`account_id`,`data_center_id`,`public_ip_address`,`device_id`,`device_type`, `network_id`);
 ALTER TABLE `cloud_usage`.`user_statistics` DROP INDEX `account_id`, ADD UNIQUE KEY `account_id`  (`account_id`,`data_center_id`,`public_ip_address`,`device_id`,`device_type`, `network_id`);
+
+INSERT INTO `cloud`.`role_permissions` (`uuid`, `role_id`, `rule`, `permission`, `sort_order`) values (UUID(), 3, 'listConfigurations', 'ALLOW', 303) ON DUPLICATE KEY UPDATE rule=rule;
```

Comment:
why is sort_order 303/304 ?

Reply:
The last entry is 302
https://github.com/apache/cloudstack/blob/main/engine/schema/src/main/resources/META-INF/db/schema-41120to41200.sql#L34

Reply:
@davidjumani
what if users have added some roles before upgrade to 4.16 ?

Reply:
Permissions can not be changed for default roles unless tweaking the DB, so IMO this is pretty safe

Reply:
@davidjumani 
I agree. I have manually tested it and approved this PR yesterday.
it looks ok to me, even if there is no change since that.